src/libm-tester/tester.c

```diff
@@ -4573,14 +4575,19 @@ int main(int argc, char **argv) {
   for(a2s=1;a2s<argc;a2s++) {
     if (strcmp(argv[a2s], "--flushtozero") == 0) {
       enableFlushToZero = 1;
+    } else if (a2s+1 < argc && strcmp(argv[a2s], "--sde") == 0) {
```

Comment:
Is the "--sde" option really needed? I am not really keen in having simulator specific options in the tester executable source code.

How about adding a wrapper script around the SDE_COMMAND that allows to invoke tester and the iutsimd executable as follows?
```
/path/to/tester /path/to/run_with_simulator.sh /path/to/iutWhateverSimd
```

Run with simulator is something I already have for SVE, it accepts one parameter in input and execute it via the simulator. Here what we could do to make it "simulator independent":

```
#!/bin/bash
SIMULATOR=$1
IUT=$2
./SIMULATOR ./IUT
```

I think that by doing so we could avoid all the changes you have done in iutsimd.c and tester.c, and the logic for handling the different simulators invocation should be less verbose in the bash script.

Reply:
The reason why I made this option is that the emulation is slow. It takes 12 minutes to test with sde, while it only takes 2 minutes without sde. With this option, the tester can check if sde is needed, and use it only if it is needed.

https://travis-ci.org/shibatch/sleef/jobs/319800736

Reply:
I see, it make sense. Thanks for the explanation.